packages/core/src/feed.ts

```diff
@@ -0,0 +1,60 @@
+import { Subject, Observable, map } from 'rxjs'
+import type { StreamMetadata, StreamState } from '@ceramicnetwork/common'
+import { CommitID } from '@ceramicnetwork/streamid'
+import type { AnchorStatus } from '@ceramicnetwork/common'
+
+export type Document = {
+  id: CommitID
+  content: any
+  anchorStatus: AnchorStatus
+  metadata: StreamMetadata
+}
+
+class DocumentsSubject extends Observable<Document> {
+  readonly subject: Subject<StreamState>
+  constructor() {
+    super((subscriber) => {
+      const subscription = this.subject
+        .pipe(
+          // transform each incoming StreamState to a Document
```

Comment:
Don't we already have a method or utility function that handles this somewhere please @ukstv @stbrody?
Looks like the kind of logic we want to have consistent over the entire codebase.

Reply:
I don't think so.  This `Document` class used here in the feed is a brand new thing.